accessibility.md

```diff
@@ -0,0 +1,10 @@
+## Accessibility report:
++ Page title - no issues found
```

Comment:
- [ ] In your <title>My Portfolio</title>, change it to a more meaningful title, such as <title>Ismail's Portfolio</title> or <title>Ismail Full Stack Developer</title>.  This will help the visually impaired with their readers or the general public to understand in a first glass the whole website.